content/docs/user-guide/interactivity/physics/_index.md

```diff
@@ -2,4 +2,8 @@
 title: Physics
 ---
 
-{{< placeholder >}}
+O3DE's support for physics simulation is built around:
```

Comment:
```suggestion
**Open 3D Engine (O3DE)** supports physics simulation with the following components.
```
O3DE needed to be spelled out and bolded, but it looked odd with the possessive.

Reply:
IMHO I would also avoid the use of the word component, unless you mean an actual O3DE component. Its an overloaded term.

Agree with changing O3DE to be spelt and bolded if thats the style. Will make that change.